### DISCUSSION TITLE
Same color of clothes

### DISCUSSION THREAD

**Discussion by Jano14** [lllyasviel/Fooocus#2426] [Q&A] [answered]
Hello! This software is incredible, but I'm running into a problem that I can't solve. I want to generate images of a girl with different colors of clothes, but it always generates images with green clothes.

To keep the face consistent I use the image prompt, adding three images of the girl's face with FaceSwap enabled. The only green thing in these images are the girl's eyes.

The prompt I give is: Medium shot, woman in jeans and [[black, red, blue]] jacket

Thank you very much for the help!

**Comment by mashb1t** [comment, marked as answer]
You can try to increase the prompt keyword weight of the colors by writing e.g. `(blue:1.5)` for 50% more weight on blue.
Wrapping the new array syntax most likely works.
Hope this helps!

**Reply by Jano14** [reply]
Hi @mashb1t,

Thanks for your quick response.

It still doesn't work at all well, I put this prompt: Medium shot, woman in jeans and [[black, red:1.7, blue]] jacket and it's true that the green stopped predominating, but it kept coming out, then the blue, some black but no red. 

Also, it stopped making her eyes green, so I changed the prompt to: Medium shot, woman with green eyes in jeans and [[black:1.1, red:1.8, blue:1.1]] jacket, with that she has green eyes but also mostly green clothes, some blue and some black but nothing red.

Can you think of anything else to ensure that the color of the eyes does not mix with the color of the clothes?

Thanks again!

**Reply by mashb1t** [reply]
The syntax is actually `([[blue,red]]:1.2)`, with parentheses surrounding the array syntax.
The keywords in the array are replaced for each iteration, which leads to `(blue:1.2)` and then `(red:1.2)` being in the prompt.
Please adjust accordingly and check again. If this doesn't help then you'll have to go for inpainting the eyes.
